gora.html

```diff
@@ -0,0 +1,168 @@
+ 	    	<h4>Jenn Gwapa</h4>
+ 	    	<p>CEO/Founder</p>
+			<p>Do not seek to change what has come before. Seek to create that which has not.</p>
+ 	      </div>
+ 	      <div class="column">
+ 	        <img class="column-photo" src="../gora/img/team/03.jpg" width="150" height="150" alt="лысый" class="flexible-image">
+ 	    	<h4>Jenn Gwapa</h4>
+ 	    	<p>CEO/Founder</p>
+ 	    	<p>Do not seek to change what has come before. Seek to create that which has not.</p>
+ 	      </div>
+ 	      <div class="column">
+ 	        <img class="column-photo" src="../gora/img/team/04.jpg" width="150" height="150" alt="черная" class="flexible-image">
+ 	    	<h4>Jenn Gwapa</h4>
+ 	    	<p>CEO/Founder</p>
+ 	    	<p>Do not seek to change what has come before. Seek to create that which has not.</p>
+ 	      </div>
+ 	    </div>
+ 	</div>
+  </div>
+</div>
+ <!-- next page================================-->
+<div class="information">
+	<div>
+ 	  <div class="title4">
+ 		<H4 class="inform2">take a look at  <b>our services</b></H4>
+ 		<div class="border-inform2"></div>
+ 	  </div>
+	    <div class="border-intro2"></div>
+ 		  <div class="sub-title4">
+ 		    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.</p>
+ 		  </div>
+ 		<div class="row2">
+ 	      <div class="column2">
+ 	    	<img class="column-photo2" src="../gora/img/4title/computer.jpg" width="150" height="150" alt="комп" class="flexible-image" >
+ 	    	<h4>web design</h4>
+ 	    	<p class="sub-text">Do not seek to change what has come before. Seek to create that which has not.</p>	
+ 	      </div>
+ 	      <div class="column2">
+ 	        <img class="column-photo2" src="../gora/img/4title/telephon.jpg"width="150" height="150" alt="тлф" class="flexible-image">
+ 	    	<h4>mobile apps</h4>
+			<p class="sub-text">Do not seek to change what has come before. Seek to create that which has not.</p>
+ 	      </div>
+ 	      <div class="column2">
+ 	         <img class="column-photo2" src="../gora/img/4title/photoaparat.png" width="150" height="150" alt="фото" class="flexible-image">
+ 	    	 <h4>photography</h4>
+ 	    	 <p class="sub-text">Do not seek to change what has come before. Seek to create that which has not.</p>
+ 	      </div>
+ 	      <div class="column2">
+ 	        <img class="column-photo2" src="../gora/img/4title/%D1%80%D1%83%D0%BF%D0%BE%D1%80.png" width="150" height="150" alt="рупор" >
+ 	    	<h4>marketing</h4>
+ 	    	<p class="sub-text">Do not seek to change what has come before. Seek to create that which has not.</p>
+ 	      </div>
+ 	    </div>	
+	</div>
+</div>
+   <!-- next page================================-->
+<div class="line"></div>
+  <div class="footer">
```

Comment:
it is `contact-us` section - not a footer
footer is below and use semantic tag pls `<footer>`